src/build.sh

```diff
@@ -11,10 +11,22 @@ fi
 set -e                          # Abort on errors
 
 
+    
+# Define some environment variables
+export CC=${EXTERNAL_CC:-${CC}}
+export CXX=${EXTERNAL_CXX:-${CXX}}
+export F90=${EXTERNAL_F90:-${F90}}
+export LD=${EXTERNAL_LD:-${LD}}
+export CFLAGS=${EXTERNAL_CFLAGS:-${CFLAGS}}
+export CXXFLAGS=${EXTERNAL_CXXFLAGS:-${CXXFLAGS}}
+export F90FLAGS=${EXTERNAL_F90FLAGS:-${F90FLAGS}}
+export LDFLAGS=${EXTERNAL_LDFLAGS:-${LDFLAGS}}
```

Comment:
ideally this should be a separate commit (and in a perfect world we would not need it of course...).